docs/spec/build.gradle

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import org.apache.tools.ant.taskdefs.condition.Os
+
+def files = ["sanitations.bal"];
```

Comment:
AFAIU the `sanitations.bal` should only be executed whenever we update the openapi-spec. So, I don't think we need to include this into the gradle build. I think we can add a gradle task which can be used to execute openapi sanitation on demand. @NipunaRanasinghe WDYT ?

Reply:
@ayeshLK please note the gradle build is added *only to build the sanitation script* during the project build, to make sure that the ballerina source code won't break with the upcoming language versions. And like you mentioned, running the script should be a one time thing, so it shouldn't be a part of the gradle build.

Reply:
Agreed +1. As an improvement we can use the Ballerina nightly docker image for this build IMO. @NipunaRanasinghe WDYT ?

Reply:
@ayeshLK it seems we currently dont support single files in the Ballerina gradle plugin. Confirmed with @ThisaruGuruge and @MohamedSabthar as well.

Btw I still believe your point is valid and ideally all the Ballerina build and run processes should go through the Ballerina gradle plugin, specially to make sure that we use the same version across all the ballerina builds within a given connector project.

Reply:
+1